app/assets/stylesheets/arclight/modules/search_results.scss

```diff
@@ -39,6 +40,7 @@
 
 .documents-compact {
   .document {
+    .al-show-breadcrumb,
```

Comment:
Is this addition needed? I'm seeing a similar class already here: https://github.com/projectblacklight/arclight/pull/586/files#diff-eb779f6d0005b800278525878e3aac87R1

Reply:
Removed now.